src/escape.rs

```diff
@@ -0,0 +1,81 @@
+//! Escape characters that may have special meaning in a shell, including spaces.
+//! This is a modified version of the [`shell-escape::unix`] module of [`shell-escape`] crate.
+//! 
+//! [`shell-escape`]: https://crates.io/crates/shell-escape
+//! [`shell-escape::unix`]: https://docs.rs/shell-escape/latest/src/shell_escape/lib.rs.html#101
+
+
+fn whitelisted(ch: char) -> bool {
```

Comment:
Let's take byte here:

```suggestion
fn whitelisted(byte: u8) -> bool {
```